GX Password/MainPage.xaml.cs

```diff
@@ -244,26 +263,44 @@ private void btClipboard_click(object sender, RoutedEventArgs e)
 		private void btOK_click(object sender, RoutedEventArgs e)
 		{
 
-			if (tsLett.IsOn == false && tsNumb.IsOn == false && tsSymb.IsOn == false)
-			{
-				P.Text = "";
+			if (tsLett.IsOn == true && tsLettUp.IsOn == false && tsLettDown.IsOn == false) {
```

Comment:
Nice.
Wouldn't it be better if you use this fragment of code every time the uppercase and lowercase options are changed?